CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Features
 
+- Attach screenshots to native errors on iOS/macOS ([#878](https://github.com/getsentry/sentry-unity/pull/878))
```

Comment:
@bitsandfoxes can you please check mac? The sample still doesn't compile on m1

Reply:
On it

Reply:
result from chate: doesn't work: `Sentry - info:: NO UIKit`